src/pooled_connection.rs

```diff
@@ -296,7 +296,7 @@ pub(crate) enum MprisMessage {
     TrackListReplaced {
         unique_name: String,
         ids: Vec<TrackID>,
-        current_id: TrackID,
+        _current_id: TrackID,
```

Comment:
This feels "wrong" to me but I cannot motivate any other change better than what we have here. :laughing: 

- Removing it since it's not used feels worse because then it's unclear that we _could_ use this information if it becomes important later - as it is sent in the underlying MPRIS event.
- Adding a flag to allow it being unused - because the theory of making unused things explicit is sound.

I think it feels wrong mostly because I assume we *should* use it somewhere but I forgot to. I suppose we should leave it with `_` prefix for now and maybe we can get to it later.